src/transformGDocs/listsGDocs.ts

```diff
@@ -0,0 +1,78 @@
+export function transformGDocsLists(html: string): string {
+    if (html.indexOf(`list-style-type:`) === -1) {
+        return html;
+    }
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(html, `text/html`);
+    let listStack: HTMLElement[] = [];
+    const listElements = doc.querySelectorAll(`[style*="list-style-type:"]`);
+    listElements.forEach(node => {
+        const el = <HTMLElement>node;
+        const arialevel: string = el.getAttribute("aria-level")!;
+        if (!arialevel) {
+            while (+arialevel > listStack.length) {
+                    const newList = createListElement(el);
+                    if (listStack.length > 0) {
+                        listStack[listStack.length - 1].appendChild(newList);
+                    } else {
+                        el.before(newList);
+                    }
+                    listStack.push(newList);
+                }
+                while (+arialevel < listStack.length) {
+                    listStack.pop();
+                }}
```

Comment:
Make sure to run prettier or something similar on your changes as we are currently do not force any code formatting rules here.